### ISSUE TITLE
A11y_.NETCore_RecipesAppMyRecipesNew_NewRecipe_Orientation: At landscape orientation "New Recipe" heading is not visible clearly.

### ISSUE THREAD

**Issue by PoojaNamde** [dotnet/maui#15317]
### Description

[Check out Accessibility Insights!](https://nam06.safelinks.protection.outlook.com/?url=https://accessibilityinsights.io/&data=02%7c01%7cv-nikpap%40microsoft.com%7cb67b2c4b646d4f9561a208d6f4b5c39b%7c72f988bf86f141af91ab2d7cd011db47%7c1%7c0%7c636965458847543876&sdata=6wiDneF0tmnVt4CroW43rkfVbB%2BL%2BAAdn0nD7xR5hQM%3D&reserved=0) - Identify accessibility bugs before check-in and make bug fixing faster and easier.

**"Please do not close this bug. This bug should only be closed by TCS, C+AI Accessibility or the Divisional Driver"**

### GitHub Tags:
#A11yMAS; #A11ySev2; #A11yTCS; #DesktopApp; #BM_.NETCore_iOS_May23; #.NET Core; #WCAG1.3.4; #iOS; #iPhone; #A11yWCAG2.1; #Benchmark;

### Environment Details:​
Application Name: Rachel's Recipe
Operating System: iOS (iPhone 6s)

### Repro Steps:

1. Install and open ".NET MAUI" app.
2. Swipe till "My Recipes" and double tap.
3. Swipe till "New" button and double tap.
4. "New Recipe" Screen will open.
5. Change to iOS in "Landscape" mode and observe that "New Recipe" heading should be clearly visible.

### Actual Results:
At landscape orientation "New Recipe" heading is not visible clearly.

**Similar issue is also observed in below scenario:**
**Issue 1: Recipes App My Recipes Edit_ Edit Recipe:**

1. Install and open ".NET MAUI" app.
2. Swipe till "My Recipes" and double tap.
3. Swipe till any existing recipe and double tap.
4. "Recipe" Screen will open.
5. Swipe till "Edit" button and double tap.
6. Change to iOS in "Landscape" mode and observe that "Edit Recipe" heading should be clearly visible.

### Expected Results:
At landscape orientation "New Recipe" heading should be clearly visible.

### User Impact:
Low vision users will not be able to see the "New Recipe" heading and will not get information about it.

### Attachment:

![IOS Issue 1](https://github.com/dotnet/maui/assets/90901519/19d6fe19-2546-4cfb-8220-9bbad9ab3876)

### Link to public reproduction project repository

Test

### Version with bug

Unknown/Other

### Last version that worked well

Unknown/Other

### Affected platforms

iOS

### Affected platform versions

iOS version 15.7.6

**Comment by PoojaNamde** [comment]
@rachelkang 

Could you please provide the latest application build to verify this issue.

**Comment by 12-shweta** [comment]
@rachelkang  could you please help us with Latest build to verify the bug

**Comment by 12-shweta** [comment]
#A11yWCAG2.1

**Comment by PoojaNamde** [comment]
@rachelkang 

Could you please help us with Latest build to verify the bug.

**Comment by rachelkang** [comment]
Unfortunately I was mistaken. This still needs to be fixed. More info at https://github.com/dotnet/maui/issues/15982

**Comment by rachelkang** [comment]
worked around by removing margin

**Comment by PoojaNamde** [comment]
#Regressed:08-17-23; #Closed;

Verified the bug in latest environment.
Bug is fixed and does not repro. Hence closing the bug.
Please find the closing attachment
